website/docusaurus.config.ts

```diff
@@ -90,10 +90,10 @@ const config: Config = {
       respectPrefersColorScheme: true,
     },
     navbar: {
-      title: 'DAMIAO',
+      // title: 'DAMIAO',
```

Comment:
Why do we need to comment out this?

Reply:
Sorry, I didn't explain enough.
I removed it 'DAMIAO' because two looked uncool. Left it commented out in case we tweak settings later.

<img width="1713" height="593" alt="Screenshot from 2025-12-03 09-45-33" src="https://github.com/user-attachments/assets/b11fa77d-51fe-41d6-8350-e5f845933fb4" />